db/migrate/20240608121440_transfer_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class TransferTransactions < ActiveRecord::Migration[7.1]
+  def change
+    create_table :table_name do |t|
+      t.integer :source_account, null: false
+      t.integer :destination_account, null: false
```

Comment:
Not making any sense. 
If you have tried to make foregin key relation here then it is not the way.